MIP/mip-84/README.md

```diff
@@ -0,0 +1,182 @@
+
+- **Description**: This MIP clarifies the trust assumptions for the Relayer, and what consequences should be drawn from these.
+- **Authors**: Andreas Penzkofer
+- **Desiderata**: [MD-74](../MIP/mip-74)
+
+## Abstract
+
+The lock/mint-type bridge has the following core actors and components: a user who initiates a transfer, a bridge operator, bridge contracts that facilitate the transfer, and a relayer that submits the completion of the transfer to the bridge contracts.
+
+The trust assumptions on the relayer component have significant implications for the security of the bridge and they can introduce the need for additional components.
+
+In a scenario based approach we clarify minimally required components and why they are needed. We distinguish between a trusted, partially trusted, and untrusted relayer (with proofs).
+
+## Motivation
+
+We need to clarify the trust assumptions for the Relayer and what consequences should be drawn from these. In many cases a rate limitation may be necessary, see also the Appendix [A1: Eigenlayer AVS](#a1-background-eigenlayer).
+
+This addresses in [MD-74](https://github.com/movementlabsxyz/MIP/blob/mip/rate-limiter-lock-mint-bridge/MD/md-74/README.md#D1) the following:
+
+- D1 : Specify the actors and their trust assumptions
+- D2 : Specify the risks and threats from components
+
+## Specification
+
+_The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174._
+
+We define the following terms:
+
+- **Trusted** : A component is trusted if it is assumed to be secure and reliable. No errors in the component can occur. Nor keys can get compromised.
+- **Partially trusted** : A component is partially trusted if it is assumed that under normal operations it is secure and reliable. However, errors may occur due to bugs, misconfigurations, or other reasons. Furthermore, the question is raised if the software component is secure. Protective measures should be taken to reduce the maximally caused damage that the component can cause incase it becomes malicious or faulty.
+- **Untrusted** : A component is untrusted if it is assumed that it is not secure and reliable. Any action should be approved by a trusted party and require a proof of correctness.
+
+### Base assumptions
+
+We make the following base assumption:
+
+- All contracts are secure and reliable.
+- The bridge operator is trusted.
+
+### Trusted Relayer
+
+**Assumption**: The relayer is fully trusted to submit the completion of the transfer to the bridge contracts.
+
+**Risks**: Since the component is trusted, in principle no risks are associated with it. The risk lies in the trust assumption. If the Relayer software component becomes malicious or faulty, the bridge is compromised, potentially leading to significant loss of funds bounds.
```

Comment:
I would remove:
 The risk lies in the trust assumption. If the Relayer software component becomes malicious or faulty, the bridge is compromised, potentially leading to significant loss of funds bounds.

Reply:
removed